### PR TITLE
Update issue templates

### LINKED ISSUE #511: Replace the existing issue templates with more "GitHuby" ones

### PR DESCRIPTION
**Pull request by jmg-duarte**
closes #511 

bug:
image

feature:
image

labels don't show up because I have a fork and the ones from this repo aren't brought along

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/01-feature.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        Thanks for taking the time to fill out this bug report!
+  - type: dropdown
+    id: product
+    attributes:
+      label: Product
+      description: Which product are you running?
+      options:
+        - Actyx
+        - Actyx CLI
+        - Node Manager
```
Comment (Kelerchian) [inline .github/ISSUE_TEMPLATE/01-feature.yaml:19]: Missing Actyx SDK

**Review by Kelerchian — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/00-bug.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: dropdown
+    id: product
+    attributes:
+      label: Product
+      description: Which product are you running?
```
Comment (rkuhn) [inline .github/ISSUE_TEMPLATE/00-bug.yaml:15]: The question is rather: which product is the bug in?

File: .github/ISSUE_TEMPLATE/01-feature.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: dropdown
+    id: product
+    attributes:
+      label: Product
+      description: Which product are you running?
```
Comment (rkuhn) [inline .github/ISSUE_TEMPLATE/01-feature.yaml:15]: same as above

**Review by rkuhn — APPROVED** [review]
some small things; we’ll iterate on these later in any case

**Pull request merged**